### ISSUE TITLE
Flashing textures

### ISSUE THREAD

**Issue by Kirussen** [CloudOfShame/BrutalOblige#5]
On the new wad i did with project brutality I got to a place where 2 textures turns to black and to normal rapidly:

Normal here:

![20170420144326_1](https://cloud.githubusercontent.com/assets/27687576/25247178/1deee40a-25d8-11e7-9cc3-c46ced0f472b.jpg)

Turning to black:

![20170420144324_1](https://cloud.githubusercontent.com/assets/27687576/25247198/30427234-25d8-11e7-896c-715df511a68e.jpg)

**Comment by CloudOfShame** [comment]
Could just be an odd lighting effect. I agree that it doesn't look good though. Any chance you could send me the wad to have a look at?

**Comment by Kirussen** [comment]
It's not really far in the levels, you should find it in 15-20 min if you start from the beginning.

[obligepb1.zip](https://github.com/CloudOfShame/BrutalOblige/files/945363/obligepb1.zip)

**Comment by CloudOfShame** [comment]
Yeah it was just a lighting effect, light strobe. I can safely say this one wasn't my fault, regular Oblige could have done the same thing.

I can see why you thought it was a glitch though, it really doesn't look good. It might be possible to replace Doom strobe lights with some fancy GZDoom effects which will look alright even if the sector isn't fully enclosed but that will require a bit of research. I'm going to mark this as an ehancement and leave it as low priority for now.

**Comment by Kirussen** [comment]
Great to know that it was already there.
